scripts/deploy_testnet_instance.py

```diff
@@ -0,0 +1,251 @@
+    print(f"> Solido program id is {solido_program_id}")
+    config["solido_program_id"] = solido_program_id
+    with open(well_known_config_location, "w") as f:
+        json.dump(config, f)
+
+    ### Multisig
+    multisig_program_id = config["multisig_program_id"]
+    if multisig_program_id is None:
+        print("\nUploading Multisig program ...")
+        multisig_program_id = solana_program_deploy(
+            get_solido_program_path() + "/serum_multisig.so"
+        )
+    else:
+        print("\nUsing existing Multisig program ...")
+    print(f"> Multisig program id is {multisig_program_id}")
+    config["multisig_program_id"] = multisig_program_id
+    with open(well_known_config_location, "w") as f:
+        json.dump(config, f)
+
+    multisig_instance = config["multisig_address"]
+    if multisig_instance is None:
+        print("\nCreating multisig instance ...")
+        multisig_data = multisig(
+            "create-multisig",
+            "--multisig-program-id",
+            multisig_program_id,
+            "--threshold",
+            "1",
+            "--owners",
+            owner.pubkey,
+            keypair_path=owner.keypair_path,
+        )
+        multisig_instance = multisig_data["multisig_address"]
+    else:
+        print("\nUsing existing multisig instance ...")
+    print(f"> Multisig instance is at {multisig_instance}")
+    config["multisig_address"] = multisig_instance
+    with open(well_known_config_location, "w") as f:
+        json.dump(config, f)
+
+    solido_address = config["solido_address"]
+    if solido_address is None:
+        print("\nCreating Solido instance ...")
+        result = solido(
+            "create-solido",
+            "--multisig-program-id",
+            multisig_program_id,
+            "--solido-program-id",
+            solido_program_id,
+            "--max-validators",
+            "9",
+            "--max-maintainers",
+            "3",
+            "--max-commission-percentage",
+            str(MAX_VALIDATION_COMMISSION_PERCENTAGE),
+            "--treasury-fee-share",
+            "5",
+            "--developer-fee-share",
+            "2",
+            "--st-sol-appreciation-share",
+            "93",
+            "--treasury-account-owner",
+            owner.pubkey,
+            "--developer-account-owner",
+            owner.pubkey,
+            "--multisig-address",
+            multisig_instance,
+            keypair_path=owner.keypair_path,
+        )
+        solido_address = result["solido_address"]
+    else:
+        print("\nUsing existing Solido instance ...")
+    print(f"> Solido instance is at {solido_address}")
+    config["solido_address"] = solido_address
+    with open(well_known_config_location, "w") as f:
+        json.dump(config, f)
+
+    solido_instance = solido(
+        "show-solido",
+        "--solido-program-id",
+        solido_program_id,
+        "--solido-address",
+        solido_address,
+    )
+
+    solana(
```

Comment:
Why we want to set upgrade authority by default?
This transaction actual only for new instance
If instance already exist, need verify that current multisig_owner have this upgrade authority access